lib/hexpm/repository/package_reports.ex

```diff
@@ -62,4 +155,38 @@ defmodule Hexpm.Repository.PackageReports do
     PackageReportComment.all_for_report(report_id)
     |> Repo.all()
   end
+
+  defp email_user_about_new_report({:ok, package_report}, user) do
+    user
+    |> Hexpm.Repo.preload(emails: [user: :emails])
```

Comment:
This looks like a strange preload. Is it not enough to only preload `:emails` once?